shared/wallets/send-form/body/advanced.tsx

```diff
@@ -63,7 +63,7 @@ const SendBodyAdvanced = (props: SendBodyAdvancedProps) => (
   <Kb.Box2 fullWidth={true} direction="vertical" style={sharedStyles.container}>
     <Kb.ScrollView style={sharedStyles.scrollView}>
       <AssetInputRecipientAdvanced />
-      <AssetPathIntermediate />
+      <AssetPathIntermediate forSEP7={false} />
```

Comment:
would it make sense to make `forSEP7` default to false since that's likely the more common case?